com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDCameraUI.Skin.cs

```diff
@@ -6,13 +6,15 @@ namespace UnityEditor.Rendering.HighDefinition
     static partial class HDCameraUI
     {
         const string generalSettingsHeaderContent = "General";
-        const string physicalSettingsHeaderContent = "Physical";
+        const string renderingSettingsHeaderContent = "Rendering";
+        const string physicalSettingsHeaderContent = "Physical Camera";
+        const string environmentSettingsHeaderContent = "Environment";
         const string outputSettingsHeaderContent = "Output";
 
         const string clippingPlaneMultiFieldTitle = "Clipping Planes";
```

Comment:
Sorry for that. Seams that we miss L10n here. Can you convert this or add a FB for those fields please?

Reply:
added localization :)